pkgs/development/libraries/mesa/default.nix

```
@@ -233,6 +234,9 @@ self = stdenv.mkDerivation {
       fi
     done
 
+    # NAR doesn't support hard links, so convert them to symlinks.
+    ${jdupes}/bin/jdupes -Hlr "$drivers"
```

Since I (and probably others) don't really know all the options:
```suggestion
    ${jdupes}/bin/jdupes --hard-links --link-soft --recurse "$drivers"
```